examples/type-guard/type_guard.bal

```diff
@@ -1,32 +1,32 @@
 import ballerina/io;
 
 public function main() {
-    // The variable `value` can hold a value of type `string` or `int` or `boolean`.
+    // The `value` variable can hold a value of the type `string`, `int` or `boolean`.
     string|int|boolean value = 10;
 
-    // The type guard can be used with `value`, to test which of the types it belongs to and perform conditional
+    // The type guard can be used with `value` to test to which of the types it belongs to and perform conditional
     // logic based on the type which it belongs to.
     if (value is string) {
-        // The type of `value` within this block is narrowed to `string`, and thus `value` can be assigned to a
-        // variable of type `string`.
+        // The type of `value` within this block is narrowed to `string`, and thus, the`value` can be assigned to a
+        // variable of the type `string`.
         string str = value;
         io:println("value is a string: ", str);
     } else if (value is int) {
-        // The type of `value` within this block is narrowed to `int`, and thus `value` can be used in a
+        // The type of `value` within this block is narrowed to `int`, and thus, the `value` can be used in a
```

Comment:
Same comment as above. Applicable for L25 and L29 too.

Reply:
+1 We don't need it here.

Reply:
```suggestion
        // The type of `value` within this block is narrowed to `int`, and thus, `value` can be used in a
```